presto-expressions/src/main/java/com/facebook/presto/expressions/LogicalRowExpressions.java

```diff
@@ -370,6 +379,101 @@ public RowExpression filterConjuncts(RowExpression expression, Predicate<RowExpr
         return combineConjuncts(conjuncts);
     }
 
+    public static GroupingElement removeGroupingElementPrefix(GroupingElement groupingElement, Optional<Identifier> removablePrefix)
```

Comment:
Can we move this utils back to ExpressionUtils (or the class you used to put these helpers, which I forgot what it was...)? `Expression` and `RowExpression` are fundamentally differently. We'd better leave expression-related helpers to expression classes and vice versa.